src/components/Page/PageHeader.tsx

```diff
@@ -6,37 +6,42 @@ import PageMenu from './PageMenu'
 import { PageHeaderProps } from './types'
 import styles from './PageHeader.module.scss'
 
-const PageHeader: FC<PageHeaderProps> = ({ withMenu = true, withSearch }) => {
+const PageHeader: FC<PageHeaderProps> = ({
+  chainConfig,
+  withSearch,
+  withMenu = true,
+}) => {
+  const links = [
+    { caption: 'Home', href: '/' },
+    { caption: 'Verify Contract', href: '/verifyContract' },
+    {
+      caption: 'API',
+      href: 'https://developers.dex.guru',
+    },
+    {
+      caption: 'Dex Metrics',
+      href: `https://dex.guru/markets?chain=${chainConfig.shortName}`,
+    },
+  ]
+  chainConfig.features.some((x) => x.name === 'csr') &&
```

Comment:
Aren't we doing this for each render?

I'd rather pass links as a component prop and resolve it via getStaticProps on page where it's used.